### PR TITLE
Moves pkg/kubectl/util/i18n to staging

### PR DESCRIPTION
**Pull request by seans3**
* Moves `pkg/kubectl/util/i18n` to staging.
* Moves translated `pkg/kubectl/generated/bindata.go` to staging.
* Updates `hack/generate-bindata.sh` to output the `bindata.go` translations to staging.
* This PR defers the long-term solution to where `/translations` will live until `kubectl` actually moves out of staging, unblocking the move to staging (every kubectl command depends on translated bindata.go).
* Manually tested `hack/generage-bindata.sh`
* Publishes this code to the new kubectl repositories: https://github.com/kubernetes/kubectl/pkg/util/i18n
https://github.com/kubernetes/kubectl/pkg/generated
* Code will be available for re-use after publishing.

/kind cleanup
/sig cli
/area kubectl
/area code-organization
/priority important-soon

```release-note
NONE
```

### CODE REVIEW

**Comment by seans3** [comment]
/assign @brendanburns 
/sig ux

This updates the location of translated `bindata.go` to allow kubectl to move into staging. It defers the long-term status of `/translations` until `kubectl` moves out of staging.

**Comment by liggitt** [comment]
update translations/README.md to point to the right path:

> Once you've added a new language, you'll need to register it in
> `pkg/kubectl/util/i18n/i18n.go` by adding it to the `knownTranslations` map.

**Comment by liggitt** [comment]
one nit on the readme, looks like a reasonable iterative step otherwise

**Comment by liggitt** [comment]
/approve
changes look good to me, will wait for readme update and give brendan/dims a chance to look and lgtm

**Comment by seans3** [comment]
> one nit on the readme, looks like a reasonable iterative step otherwise

Done

File: hack/generate-bindata.sh [hunk 1]
```diff
@@ -65,7 +65,7 @@ fi
 rm -f "${BINDATA_OUTPUT}.tmp"
 
 # These are files for runtime code
-BINDATA_OUTPUT="pkg/kubectl/generated/bindata.go"
+BINDATA_OUTPUT="staging/src/k8s.io/kubectl/pkg/generated/bindata.go"
```
Comment (BenTheElder) [inline hack/generate-bindata.sh]: I assume this will eventually move to something run in k8s.io/kubectl?

**Review by BenTheElder — APPROVED** [review]
/lgtm

File: hack/generate-bindata.sh (hunk 1 above)
Comment (liggitt) [reply hack/generate-bindata.sh]: exactly

**Pull request merged**